scipy/special/stirling2.h

```diff
@@ -0,0 +1,119 @@
+#ifndef STIRLING_H
+#define STIRLING_H
+
+#include <complex.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "_lambertw.h"
```

Comment:
```suggestion
#include "special/lambertw.h"
```

Sorry, the structure for the new C++ implementations changed here https://github.com/scipy/scipy/pull/19515.

Reply:
no worries, let's do that then :)

Reply:
hmm, that didn't do it. It looks like this change ( #19515 ) was merged after I did the rebase last weekend-rebasing again to confirm that fixes.